internal/cephfs/store/backingsnapshot.go

```diff
@@ -170,3 +170,70 @@ func UnrefSelfInSnapshotBackedVolumes(
 		},
 	)
 }
+
+func UpdateRefCount(
+	ctx context.Context,
+	volOptions *VolumeOptions,
+	clusterName string,
+	setMetadata bool,
+	secrets map[string]string,
```

Comment:
These variables are not used in this function.

Reply:
Yes, updated.
Thanks!